celery/app/amqp.py

```diff
@@ -309,7 +309,8 @@ def as_task_v2(self, task_id, name, args=None, kwargs=None,
                    time_limit=None, soft_time_limit=None,
                    create_sent_event=False, root_id=None, parent_id=None,
                    shadow=None, chain=None, now=None, timezone=None,
-                   origin=None, argsrepr=None, kwargsrepr=None):
+                   origin=None, argsrepr=None, kwargsrepr=None,
+                   **compat_kwargs):
```

Comment:
Thanks for applying the suggested changes. Although it would work, you don't need `**compat_kwargs` in this signature, so I would recommend to perform as little changes as possible.

Reply:
Sure thing - PR updated!